src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -854,6 +854,12 @@
           long-summary: |
               Used to control the mode the network plugin should operate in. For example, "overlay" used with
               --network-plugin=azure will use an overlay network (non-VNET IPs) for pods in the cluster.
+        - name: --network-policy
+          type: string
+          short-summary: Update the mode of a network policy.
+          long-summary: |
+              Specify "azure" for Azure network policy manager, "cilium" for Azure CNI Overlay powered by Cilium, and "none" to uninstall "azure" network policy.
+              Defaults to "" (network policy disabled).
```

Comment:
I'd suggest removing references to "none" for now, since it's not fully implemented in AKS yet.

Reply:
Removed mentioning "none"